test/controllers/BankDetailsControllerSpec.scala

```diff
@@ -718,36 +748,6 @@ class BankDetailsControllerSpec extends ItSpec {
         )
       }
 
-    "show correct error message when bars verify-personal responds with accountExists is No" in
-      new BarsFormErrorSetup("accountDoesNotExist", typeOfAccount = TypesOfBankAccount.Personal) {
-        testFormError(controller.enterBankDetailsSubmit)(validForm: _*)(expectedContentAndHref)
-
-        BarsStub.ValidateStub.ensureBarsValidateCalled(validForm)
-        BarsStub.VerifyPersonalStub.ensureBarsVerifyPersonalCalled(validForm)
-        BarsStub.VerifyBusinessStub.ensureBarsVerifyBusinessNotCalled()
-        BarsVerifyStatusStub.ensureVerifyUpdateStatusIsCalled()
-      }
-
-    "show correct error message when bars verify-business responds with accountExists is No" in
-      new BarsFormErrorSetup("accountDoesNotExist", typeOfAccount = TypesOfBankAccount.Business) {
-        testFormError(controller.enterBankDetailsSubmit)(validForm: _*)(expectedContentAndHref)
-
-        BarsStub.ValidateStub.ensureBarsValidateCalled(validForm)
-        BarsStub.VerifyBusinessStub.ensureBarsVerifyBusinessCalled(validForm)
-        BarsStub.VerifyPersonalStub.ensureBarsVerifyPersonalNotCalled()
-        BarsVerifyStatusStub.ensureVerifyUpdateStatusIsCalled()
-      }
-
-    "show correct error message when bars validate response is 400 sortCodeOnDenyList" in
```

Comment:
I don't know what the hell is going on, I haven't removed these, they're still in the spec in intellij on my local

Reply:
has it maybe moved? is it just showing where it used to be?

Reply:
Oh yeah, you're right, I panicked then 😂